cmd/transport_security_state_static_generate/transport_security_state_static_generate.go

```diff
@@ -62,13 +62,15 @@ type pinset struct {
 }
 
 type hsts struct {
-	Name                 string `json:"name"`
-	Subdomains           bool   `json:"include_subdomains"`
-	SubdomainsForPinning bool   `json:"include_subdomains_for_pinning"`
-	Mode                 string `json:"mode"`
-	Pins                 string `json:"pins"`
-	ExpectCT             bool   `json:"expect_ct"`
-	ExpectCTReportURI    string `json:"expect_ct_report_uri"`
+	Name                  string `json:"name"`
+	Subdomains            bool   `json:"include_subdomains"`
```

Comment:
@dadrian a random thing to think about, do we want to support includeSubdomains for Expect-Staple? We explicitly decided not to support includeSubdomains for Expect-CT, but for reasons that don't necessarily apply to OCSP. For Expect-CT, we are initially sending all reports to Google, and includeSubdomains seemed like a big footgun for people to accidentally leak secret subdomains to Google. (Whereas for Expect-Staple, where reports are going to the sites themselves, includeSubdomains might be useful and less footgunny.)

If we do want to support it, I think this include_subdomains flag only applies for HSTS, do you know if that's correct? We might need to introduce a separate SubdomainsForExpectStaple.